data/json/effect_on_condition.json

```diff
@@ -98,4 +98,58 @@
     "id": "EOC_MINOR_SLEEP",
     "effect": [ { "message": "You feel sleepy…" }, { "u_mod_fatigue": 20 } ]
   }
+  {
+  "type": "effect_on_condition",
+    "id": "night_messages",
+    "recurrence_min": "4 hours",
+    "recurrence_max": "8 hours",
+    "condition": {
+      "and": [
+        { "one_in_chance": 3 },
+        { "not": "is_day" },
+        "u_can_see",
+        { "is_weather": "clear" },
+        { "not": { "days_since_cataclysm": 14 } },
+        "u_is_outside"
+      ]
+    },
+    "deactivate_condition": { "days_since_cataclysm": 14 },
+    "effect": [ { "message": "NIGHT_MESSAGES_1", "snippet": true } ]
```

Comment:
This line somehow confuse the translation extraction script, either change its index name or fix the python.